aarch64_linux/aarch64_wheel_ci_build.py

```diff
@@ -110,6 +110,9 @@ def parse_arguments():
     print("Applying mkl-dnn patch to fix crash due to /sys not accesible")
     os.system("cd /pytorch/third_party/ideep/mkl-dnn && patch -p1 < /builder/mkldnn_fix/fix-xbyak-failure.patch")
 
+    print("Applying mkl-dnn patch to improve torch.compile() perf")
+    os.system("cd /pytorch/third_party/ideep/mkl-dnn && patch -p1 < /builder/mkldnn_fix/onednn-pr1768-aarch64-add-acl-sbgemm-inner-product-primitive.patch")  # noqa: E501
```

Comment:
Not to author, reviewer: I don't know why those were accepted initially, but one should avoid using `os.system` as much as possible, as it gives no signal back on whether the underlying command succeeded or failed, this way making regression detection a much harder process. (i.e. one can add `os.system("fokjhdsfkjhsef");` anywhere in this script and it will be just a no-op)

`subprocess.check_call` is a better way of doing that, for example like that:
https://github.com/pytorch/builder/blob/ab5fc904118395902a29ba74df20ca2a5e937694/aarch64_linux/aarch64_wheel_ci_build.py#L111-L112

This way typo in patch name or failure to apply the patch will result in the failure

Reply:
Sounds good we should probably change it in both palces line 111 and 114